### PR TITLE
Prom native ceph

### PR DESCRIPTION
**Pull request by hecomp**
- Prom native ceph

### CODE REVIEW

File: collector/prom-native-ceph/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  ```
+  docker exec mon1 ceph --cluster ceph mgr module disable prometheus
+  ```
+* Check Ceph Health
+  ```
+  docker exec mon1 ceph --cluster ceph -s
+  ```
+* Stop the cluster
+  ```
+  docker-compose down
```
Comment (nslaughter) [inline collector/prom-native-ceph/README.md:37]:
```suggestion
  docker compose down
```
-- Just for consistency in the commands. Both ways are valid.

File: collector/prom-native-ceph/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- Prometheus Module provides a Prometheus exporter to pass on Ceph performance counters from the collection point in ceph-mgr. Ceph-mgr receives MMgrReport messages from all MgrClient processes (mons and OSDs, for instance) with performance counter schema data and actual counter data, and keeps a circular buffer of the last N samples. This module creates an HTTP endpoint (like all Prometheus exporters) and retrieves the latest sample of every counter when scraped.
+
+```sh
+$ sudo ceph mgr module enable prometheus
+```
+```sh
+$ sudo ceph mgr module disable prometheus
+```
+
+Enabling Prometheus Output
```
Comment (nslaughter) [inline collector/prom-native-ceph/README.md:60]:
```suggestion
## Enabling Prometheus Output  
```
This looks like it's meant to be a heading to me. I believe that this is the appropriate level, but check it out with what you intended.

File: collector/prom-native-ceph/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+## Additional information
+
+- [OpenTelemetry Collector Prometheus Receiver][otel-prom-receiver]
+- [Ceph Telemetry Reference][ceph-docs-prometheus]
+
+[ls-docs-access-token]: https://docs.lightstep.com/docs/create-and-manage-access-tokens
+[ls-docs-dashboards]: https://docs.lightstep.com/docs/create-and-manage-dashboards
+[otel-prom-receiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver
+[ceph-docs-prometheus]: https://docs.ceph.com/en/quincy/mgr/prometheus/
+[learn-consul-repo]: https://github.com/hashicorp/learn-consul-docker
```
Comment (nslaughter) [inline collector/prom-native-ceph/README.md:98]:
```suggestion
```
I think this is a holdout from copy pasta.

**Review by nslaughter — APPROVED** [review]
I'm marking approve as I do believe the quality is sufficient. 

However, please look at my suggestions and consider the questions below as I think they're opportunities for improving.

Some questions to consider...
(1) do you know the load order, so you could use the `depends_on` attribute more extensively and avoid erroneous errors which really just mean a dependent container came up faster than its dependencies
(2) can you write any healthchecks (https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck) that make timings even more explicit?

Sometimes those are fast and easy things to improve. If that's so in this case then please go ahead.

File: collector/prom-native-ceph/README.md (hunk 3 above)
Comment (hecomp) [reply collector/prom-native-ceph/README.md:98]: Yes, removing.

**Comment by hecomp** [comment]
> I'm marking approve as I do believe the quality is sufficient.
> 
> However, please look at my suggestions and consider the questions below as I think they're opportunities for improving.
> 
> Some questions to consider... (1) do you know the load order, so you could use the `depends_on` attribute more extensively and avoid erroneous errors which really just mean a dependent container came up faster than its dependencies (2) can you write any healthchecks (https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck) that make timings even more explicit?
> 
> Sometimes those are fast and easy things to improve. If that's so in this case then please go ahead.

Agree. addressed.

**Pull request merged**